cmd/notary/tuf.go

```diff
@@ -546,3 +587,30 @@ func getRemoteTrustServer(config *viper.Viper) string {
 	}
 	return defaultServerURL
 }
+
+func getTrustPinning(config *viper.Viper) (trustpinning.TrustPinConfig, error) {
```

Comment:
I have the `TestConfigFileTrustPinning` test in `main_test.go`, which tests against this method and parsing the config file itself -- I put it there because we have other configuration parsing tests in that file.  If there are any particular test cases you had in mind I can definitely add them